packaging/app.organicmaps.desktop.metainfo.xml

```diff
@@ -28,6 +28,9 @@
     </p>
     <p>## Features</p>
     <p>
+      The Linux version of Organic Maps has not reached feature parity
```

Comment:
Probably best to call it the 'Desktop version' as it's also usable on MacOS

Reply:
> Probably best to call it the 'Desktop version' as it's also usable on MacOS

To the best of my knowledge appstream metadata is not usable on MacOS, and I believe only Linux users will read this. And my understanding is that creating a convergent UI which is not for desktop only is [at least still on the talbe](https://github.com/organicmaps/organicmaps/discussions/4408). And this metainfo would be suitable for that hypothetical scenario as well.

Reply:
alright fair enough :)